fbw-common/src/wasm/systems/systems_wasm/src/anti_ice.rs

```diff
@@ -0,0 +1,77 @@
+#[cfg(not(target_arch = "wasm32"))]
+use crate::msfs::legacy::execute_calculator_code;
+#[cfg(target_arch = "wasm32")]
+use msfs::legacy::execute_calculator_code;
+use systems::shared::to_bool;
+
+use crate::{ExecuteOn, MsfsAspectBuilder, Variable};
+use std::error::Error;
+
+pub(super) fn engine_anti_ice(
+    engine_count: usize,
+) -> impl FnOnce(&mut MsfsAspectBuilder) -> Result<(), Box<dyn Error>> {
+    move |builder: &mut MsfsAspectBuilder| {
+        for engine_number in 1..=engine_count {
+            builder.on_change(
+                ExecuteOn::PostTick,
+                vec![
+                    Variable::named(&format!(
+                        "BUTTON_OVHD_ANTI_ICE_ENG_{}_POSITION",
+                        engine_number
+                    )),
+                    Variable::aircraft("ENG ANTI ICE", "bool", engine_number),
+                ],
+                Box::new(move |prev_values, new_values| {
+                    let was_eng_anti_ice_push_button_on = to_bool(prev_values[0]);
+                    let is_eng_anti_ice_push_button_on_now = to_bool(new_values[0]);
```

Comment:
```suggestion
                    let is_eng_anti_ice_push_button_on = to_bool(new_values[0]);
```
now isnt needed since it is alread `is`

Reply:
Done